### PR TITLE
Feat: Create database migration

### PR DESCRIPTION
**Pull request by joaoAraki27**

### CODE REVIEW

File: backend/backend/migrations/__pycache__/0001_initial.cpython-312.pyc
Comment (FilippoQuattrocchi) [inline backend/backend/migrations/__pycache__/0001_initial.cpython-312.pyc:1]: We don't need __pycache__

File: backend/backend/migrations/__pycache__/__init__.cpython-312.pyc
Comment (FilippoQuattrocchi) [inline backend/backend/migrations/__pycache__/__init__.cpython-312.pyc:1]: We don't need pycache

File: backend/backend/settings.py
Comment (FilippoQuattrocchi) [inline backend/backend/settings.py:1]: Works without 'backend'?

**Review by FilippoQuattrocchi — CHANGES_REQUESTED** [review]

File: backend/backend/migrations/__pycache__/0001_initial.cpython-312.pyc
Comment (joaoAraki27) [reply backend/backend/migrations/__pycache__/0001_initial.cpython-312.pyc:1]: pycache just appeared, can I delete it?

File: backend/backend/settings.py
Comment (joaoAraki27) [reply backend/backend/settings.py:1]: docker-compose exec backend python manage.py makemigrations  

This is the command that worked to create the migrations, if I hadn't created the backend or didn't specify it in the command, manage.py wouldn't find it ' service' inside the container. 
I don't know if there is another way but this is what I found to work migrations yesterday

File: backend/backend/migrations/__pycache__/0001_initial.cpython-312.pyc
Comment (FilippoQuattrocchi) [reply backend/backend/migrations/__pycache__/0001_initial.cpython-312.pyc:1]: yes

File: backend/backend/settings.py
Comment (FilippoQuattrocchi) [reply backend/backend/settings.py:1]: OK!

**Review by FilippoQuattrocchi — APPROVED** [review]

**Pull request merged**